docs/rounding.md

```diff
@@ -0,0 +1,54 @@
+## Rounding ZonedDateTimes
+
+Rounding operations (`floor`, `ceil`, and `round`) on `ZonedDateTime`s are performed in a
+[similar manner to `DateTime`](http://julia.readthedocs.org/en/latest/manual/dates/#rounding)
+and should generally behave as expected. When `VariableTimeZone` transitions are involved,
+however, unexpected behaviour may be encountered.
+
+Instead of performing rounding operations on the `ZonedDateTime`'s internal UTC `DateTime`,
+which would be computationally less expensive, rounding is done in the local time zone.
+This ensures that rounding behaves as expected and is maximally meaningful. (Consider how
+rounding to the nearest day would be resolved for non-UTC time zones, or even rounding to
+the nearest hour for time zones like `America/St_Johns` or `Australia/Eucla`.)
```

Comment:
The reader doesn't have context to why these last two time zones are special. Consider using them in an example or at least explain why they are problematic.